Mage.Sets/src/mage/cards/t/TegwyllDukeOfSplendor.java

```diff
@@ -0,0 +1,73 @@
+package mage.cards.t;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.DiesCreatureTriggeredAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.effects.common.DrawCardSourceControllerEffect;
+import mage.abilities.effects.common.LoseLifeSourceControllerEffect;
+import mage.abilities.effects.common.continuous.BoostControlledEffect;
+import mage.abilities.keyword.DeathtouchAbility;
+import mage.abilities.keyword.FlyingAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.filter.common.FilterControlledCreaturePermanent;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.filter.predicate.mageobject.AnotherPredicate;
+
+import java.util.UUID;
+
+/**
+ * @author Susucr
+ */
+public final class TegwyllDukeOfSplendor extends CardImpl {
+
+    private static final FilterCreaturePermanent filter =
+            new FilterCreaturePermanent(SubType.FAERIE, "Faeries you control");
+    private static final FilterControlledCreaturePermanent filter2 =
+            new FilterControlledCreaturePermanent(SubType.FAERIE, "another Faerie you control");
```

Comment:
this shouldn't be a creature filter as it should trigger for cards like [[bitterblossom]]

Reply:
Oh right, I always forgot Tribal for those effects.